gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,650 @@
+= Getting Started with Flex Gateway on a Kubernetes Cluster
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:version-helm: 3.0.0
+
+Get started with Anypoint Flex Gateway on a Kubernetes cluster. Set up a Flex Gateway deployment that connects to Anypoint Platform, and use the deployment to route traffic to your API. This guide covers the following high-level tasks:
+
+* Downloading, registering, and deploying Flex Gateway in Connected Mode. 
+* Creating a simple API in Anypoint API Manager that runs on your gateway. 
+* Protecting the API with a basic authentication policy that you configure for it in API Manager. 
+
+The Flex Gateway deployment process requires a working Kubernetes cluster that you create or that is provisioned for you by a Kubernetes professional. 
+
+Setting up a Kubernetes cluster requires a level of skill and expertise that is _beyond the scope of this guide to provide_. Cluster configuration in Kubernetes differs markedly by service provider and platform. For cluster configuration questions, consult documentation from your Kubernetes service provider or use the https://kubernetes.io/docs/setup/[Kubernetes documentation^]. 
+
+If you lack a Kubernetes cluster but want to try out Flex Gateway quickly, without creating a cluster, see xref:flex-gateway-getting-started.adoc[Getting Started with Flex Gateway] (on Docker). That guide uses Docker exclusively and does not require you to perform any cluster configuration.
+
+== Before You Begin
+
+Before starting the procedures in this guide, ensure that the following prerequisites are in place:
+ 
+* A username and password for your Anypoint Platform organization. If you do not have an Anypoint Platform account, https://anypoint.mulesoft.com/login/signup[create a Trial organization^] on Anypoint Platform.
+//TODO: VERIFY Manage Servers and Read Servers permissions in RTM NO LONGER NEEDED
+
+//os prereqs
+* A Linux or MacOS machine with an x86 processor. M1 and M2 processors are not supported.
```

Comment:
machine -> system?

Reply:
we're talking OS's here

Reply:
for now, leaving for consistency with the other getting started guide